addons/account/tests/test_payment_term.py

```diff
@@ -403,3 +403,33 @@ def test_payment_term_percent_round_calculation(self):
                 Command.create({'value_amount': 16.8, 'value': 'percent', 'nb_days': 0, }),
             ],
         })
+
+    def test_payment_term_labels(self):
+        # create a payment term with 40% now, 30% in 30 days and 30% in 60 days
+        multiple_installment_term = self.env['account.payment.term'].create({
+            'name': "test_payment_term_labels",
+            'line_ids': [
+                Command.create({'value_amount': 40, 'value': 'percent', 'nb_days': 0, }),
+                Command.create({'value_amount': 30, 'value': 'percent', 'nb_days': 30, }),
+                Command.create({'value_amount': 30, 'value': 'percent', 'nb_days': 60, }),
+            ],
+        })
+        # create an invoice with standard payment term
+        invoice = self.init_invoice('out_invoice', products=self.product_a)
+        invoice.invoice_payment_term_id = self.env.ref('account.account_payment_term_immediate')
```

Comment:
Generally we don't ever depend on demo/generic data in tests. Please use something available in the test setup, or create a new payment term yourself